src/codeflare_sdk/cluster/cluster.py

```diff
@@ -541,6 +541,7 @@ def from_k8_cluster_object(
             mcad=mcad,
             write_to_file=write_to_file,
             verify_tls=verify_tls,
+            local_queue=rc["metadata"]["labels"]["kueue.x-k8s.io/queue-name"],
```

Comment:
```suggestion
            local_queue=rc["metadata"].get("labels", dict()).get("kueue.x-k8s.io/queue-name", None),
```

Reply:
Thanks will update it tomorrow, but can a `RayCluster` even exists without `local_queue`? I thought you can't create one without it so label should always exists?

Reply:
Not via the SDK for sure, but I'm not positive we can guarantee that every RayCluster has the label. I would need to think on it. I think it's better to be safe than sorry here mainly.